static/app/views/settings/organization/navigationConfiguration.tsx

```diff
@@ -100,4 +100,112 @@ const organizationNavigation: NavigationSection[] = [
   },
 ];
 
+export const organizationNavigationWithAuthTokens: NavigationSection[] = [
```

Comment:
Did we just duplicate the entire nav tree here?

Reply:
Yes, I figured for this experimental phase this is fine/least likely to break? Would you prefer we mutate this in the feature flag case?